README.md

```diff
@@ -73,9 +68,12 @@ Different versions of the example scripts show different ways of describing the
 ##### Starting the buildd daemon:
 
 ```
-buildd-standalone --debug --root /var/lib/buildkit
+buildd --debug --root /var/lib/buildkit
 ```
 
+By default, the OCI/runc worker is used.
+You can set `--oci-worker=false --containerd-worker=true` to use the containerd worker.
```

Comment:
Not this PR: should we add `--prefer-containerd-worker` to make this case nicer?

Reply:
`--containerd-worker-priority=0 --oci-worker-priority=1`?

Reply:
The numbers don't feel user-friendly to me. `--worker-priority=containerd` `--worker-priority=containerd,oci` ? The ones that user defines have higher priority and order from user. Ones that are not mentioned get default order.